app/controllers/githook_controller.rb

```diff
@@ -38,9 +42,17 @@ def pull_master
   end
 
   def authenticate
-    flash[:error] = t("the_role.access_denied")
-    redirect_to(:back) unless (current_user) && (current_user.moderator?(:val))
+    flash[:error] = t('the_role.access_denied')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.